### ISSUE TITLE
Error when starting from Start-Menu

### ISSUE THREAD

**Issue by FranekStark** [geluk/pass-winmenu#61]
If I start the Application by searching for it at the Start menu and then launching it. It crashes with tht failure: 
![image](https://user-images.githubusercontent.com/22654048/81595112-3926bd00-93c2-11ea-8af9-72c53028f727.png)

But when I 'right-click' on it and select 'Open Folder' (Or something like that cause I'm not really sure what it is called on english computers) and then launching the appilcation from that opened folder it works.

**Comment by geluk** [comment]
Interesting, seems almost as if the application gets launched from two separate working directories. It reads the configuration file from the current working directory, so if those differ, it's possible that there are two configuration files. One with a syntax error, and one without.

I was under the impression that if you launch a program from the start menu, its working directory gets set to the location of the executable. In your case that does not seem to be true since the config file next to the executable is clearly fine, otherwise starting it directly from Explorer would result in the same error.

Because of this I wonder if the Start menu link launches the application with a different working directory. If so, there's most likely a broken file in there. I have created a build that shows the location of the configuration file in this error message, which should at least help you figure out which file it's talking about.

If the file mentioned in the error ends up being in a strange place (being anywhere that isn't next to `pass-winmenu.exe`), I would be interested in hearing about it.

The executable from the test build can be used as a drop-in replacement for the original pass-winmenu, you can download it here: https://ci.appveyor.com/api/buildjobs/qemo3j6thh1ebauu/artifacts/pass-winmenu%2Fbin%2FRelease%2Fpass-winmenu.exe

**Comment by FranekStark** [comment]
Okay,
I tested it with that build. And it shows a strange behaviour. It isn't starting in both cases anymore.

If I launch it from the filebrowser it says:
![image](https://user-images.githubusercontent.com/22654048/81754959-be3fce00-94b7-11ea-8aa9-1f1fddd567d2.png)
The exception text is German and means: "The Folder Name is invalid".
The logfile says:
```
[01:18:53.974] [DBG] Starting pass-winmenu v1.10.1-14-gdffaffb
[01:18:53.974] [DBG] ------------------------------
[01:18:53.977] [DBG] Enabled security protocols: Tls, Tls11, Tls12, Tls13
[01:18:54.181] [ERR] Could not start pass-winmenu: An exception occurred.
[01:18:54.186] ArgumentException (Der Verzeichnisname  ist ungültig.) in :0:0
[01:18:54.186]   -> in System.IO.FileSystemWatcher..ctor -- :0:0
[01:18:54.186]   -> in PassWinmenu.Configuration.ConfigManager.EnableAutoReloading -- :0:0
[01:18:54.186]   -> in PassWinmenu.Program.LoadConfigFile -- :0:0
[01:18:54.186]   -> in PassWinmenu.Program.Initialise -- :0:0
[01:18:54.186]   -> in PassWinmenu.Program.Start -- :0:0
```



If I launch it from Start menu it says:
![image](https://user-images.githubusercontent.com/22654048/81754858-7ae55f80-94b7-11ea-8201-f2d430a4e073.png)

The strange thing is that at this location a config file exsits. Im not sure why. I've never put it there. For installation I use chocolatey. 

Afterwords it tells me, that i has not the rights to write a logfile on that location "C:Windows\system32\pass-winmenu.log"

I've fixed this config File. Now its commin up with the first message (as starting from file browser)

So it seems that your are correct. Starting it from the startmenu will end up in another working directory.

**Comment by geluk** [comment]
Interesting finding. That is rather unexpected. Can you check if the working directory for the start menu shortcut is perhaps different from its default? If you go to 'Open File location' (screenshotted just to verify we're talking about the same thing here):
![image](https://user-images.githubusercontent.com/1516985/81755949-96059e80-94ba-11ea-8515-d67b1590d21c.png)

You should end up in a folder containing a shortcut to pass-winmenu. If you right click that and go to properties, what does the 'Start in:' field say? Does it point to system32 by any chance?

Regarding the error in the new build, that's due to an oversight on my part in the test build - if you replace the test build with v1.10.1 again it will be gone.

**Comment by FranekStark** [comment]
It seems to be not as yu are expecting. The folder which is opening is not this Start menu link folder.
It opens directly the folder where the aplication seems to be installed. 

`C:\tools\pass-winmenu`

![image](https://user-images.githubusercontent.com/22654048/81801361-97fc4b80-9514-11ea-82b7-5ce60bc96355.png)

**Comment by geluk** [comment]
That's odd. When you click 'Pin to Start' on a program it's supposed to add a shortcut to the program to `%appdata%\Microsoft\Windows\Start Menu\Programs`. Perhaps unpinning and re-pinning fixes it?

**Comment by FranekStark** [comment]
I've never pinned it. I think Windows is finding it by its file directory.

**Comment by geluk** [comment]
I suppose that means when you open a program found through the search function in Windows, it will launch it with `system32` as its working directory. I'm going to have think about how I want to resolve this. For now, the easy solution to your problem would be manually adding it to the Start menu, but it'd be nice if that wouldn't be necessary.

**Comment by geluk** [comment]
Fixed in [v1.11](https://github.com/geluk/pass-winmenu/releases/tag/v1.11). Should you still encounter any issues, please let me know!
